src/components/AppFooter/AppFooter.tsx

```diff
@@ -22,6 +22,8 @@ const AppFooter = () => {
     const currentLang = LocalStorageUtils.getValue<string>('i18n_language') || 'EN';
 
     const openLanguageSettingModal = () => showModal('DesktopLanguagesModal');
+    const origin = window.location.origin;
+    const isProduction = origin === URLConstants.derivP2pProduction;
```

Comment:
tried using proccess.env.NODE_ENV but that didn't work, this can be used since this is done in url.ts `getServerInfo` function which sets the localstorage values for appid and serverurl